clang/lib/DPCT/DPCT.cpp

```diff
@@ -826,6 +826,23 @@ int runDPCT(int argc, const char **argv) {
 
     Tool.mapVirtualFile(SourcePathList[0], SourceCode);
 
+    static const std::string UnsupportedStr{"// UNSUPPORTED:"};
```

Comment:
Can we use constexpr StringRef UnsupportedStr{"// UNSUPPORTED:"}; instead of a static const string?